testfiles/PythonPluginReplaceTraditionalManagers_LargeOffice.idf

```diff
@@ -8322,11 +8322,11 @@
 !  Seasonal-Reset-Supply-Air-Temp-Sch,  !- Schedule Name
 !  VAV_1_CoolC-VAV_1_HeatCNode;  !- Setpoint Node or NodeList Name
 
-  EnergyManagementSystem:Actuator,
-    VAV_1_SAT_setpoint,      !- Name
-    VAV_1 Supply Equipment Outlet Node,  !- Actuated Component Unique Name
-    System Node Setpoint,    !- Actuated Component Type
-    Temperature Setpoint;    !- Actuated Component Control Type
+!  EnergyManagementSystem:Actuator,
```

Comment:
@Myoldmopar should these old EMS objects be removed?

Reply:
Maybe, but it's not worth another commit and round of CI.